### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by a1346054**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    OTPClient - Simple GTK+ software to generate OTPs
-    Copyright (C) 2019 Paolo Stivanin
+    <one line to give the program's name and a brief idea of what it does.>
```
Comment (paolostivanin) [inline LICENSE:634]: You should keep the original text, just update the year

**Review by paolostivanin — CHANGES_REQUESTED** [review]
Hello,
thanks! Some things should be changed though:

1. please sign the commits
2. the License file should include the program's name and copyright year. Please do not replace those fields.

File: LICENSE (hunk 1 above)
Comment (a1346054) [reply LICENSE:634]: This is just the part of 'How to Apply These Terms to Your New Programs' section of the GPL license, it is not meant to be edited. The whole GPL license is supposed to be distributed verbatim anyway.

Explanation here: https://www.gnu.org/licenses/gpl-faq.html#GPLOmitPreamble

**Comment by a1346054** [comment]
Signed commits and clarified the license situation.

**Pull request merged**